### PR TITLE
feat(zeebe): document new flow control configuration

### PR DESCRIPTION
**Pull request by lenaschoenburg**
## Description

Documents new configuration added in https://github.com/camunda/camunda/issues/17547.
Closes https://github.com/camunda/camunda/issues/20275

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/self-managed/zeebe-deployment/configuration/broker.md [hunk 1]
```diff
@@ -512,6 +512,149 @@ threads:
...
+| minLimit     | The minimum limit. This setting can also be overridden using the environment variable `ZEEBE_BROKER_FLOWCONTROL_REQUEST_GRADIENT_MINLIMIT`.                                                                                                                                                                                                                  | 10            |
+| initialLimit | The initial limit to be used when the broker starts. The limit will be reset to this value when the broker restarts. This setting can also be overridden using the environment variable `ZEEBE_BROKER_FLOWCONTROL_REQUEST_GRADIENT_INITIALLIMIT`.                                                                                                            | 20            |
+| rttTolerance | Tolerance for changes from minimum latency. A value >= 1.0 indicating how much change from minimum latency is acceptable before reducing the limit. For example, a value of 2.0 means that a 2x increase in latency is acceptable. This setting can also be overridden using the environment variable ZEEBE_BROKER_FLOWCONTROL_REQUEST_GRADIENT_RTTTOLERANCE | 2.0           |
```
Comment (rodrigo-lourenco-lopes) [inline docs/self-managed/zeebe-deployment/configuration/broker.md:598]: 🔧  Its missing the quotes here in ZEEBE_BROKER_FLOWCONTROL_REQUEST_GRADIENT_RTTTOLERANCE.

File: docs/self-managed/zeebe-deployment/configuration/broker.md [hunk 2]
```diff
@@ -512,6 +512,149 @@ threads:
...
+| acceptableBacklog | The number of records that can be in the exporting backlog. The write rate is throttled such that the backlog stabilizes around this value when exporting is a bottleneck. This setting can also be overridden using the environment variable `ZEEBE_BROKER_FLOWCONTROL_WRITE_THROTTLING_ACCEPTABLEBACKLOG`. | 100000        |
+| minimumLimit      | The minimum write limit that is guaranteed even when throttling. This setting can also be overridden using the environment variable `ZEEBE_BROKER_FLOWCONTROL_WRITE_THROTTLING_MINIMUMLIMIT`.                                                                                                                | 100           |
+| resolution        | The frequency at which the throttling is updated. This setting can also be overridden using the environment variable `ZEEBE_BROKER_FLOWCONTROL_WRITE_THROTTLING_RESOLUTION`.                                                                                                                                 | 15s           |
+
```
Comment (rodrigo-lourenco-lopes) [inline docs/self-managed/zeebe-deployment/configuration/broker.md:657]: 🔧  we are also missing the snipet for the throttling

**Review by rodrigo-lourenco-lopes — APPROVED** [review]
Thanks for adding the docs for flow control 🙌  Just two minor fixes before merge 🚀

**Review by christinaausley — APPROVED** [review]
Approving but CC @conceptualshark for awareness and any additional SM review 😄

**Review by rodrigo-lourenco-lopes — APPROVED** [review]
Thanks for the fix! 🚀

**Pull request merged**